helpers/helpers.html

```diff
@@ -0,0 +1,44 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+
+<script>
+  Polymer.AppLayout = Polymer.AppLayout || {};
+
+  Polymer.AppLayout.scrollEffects = Polymer.AppLayout.scrollEffects || {};
+
+  /**
+   * Helper function that scrolls to a particular
+   * set of coordinates in a scroll target without scroll effects transitions.
+   *
+   * @param {number} scrollLeft
+   * @param {number} scrollTop
+   * @param {HTMLElement?} scrollTarget
+   */
+  Polymer.AppLayout.scrollTo = function scrollTo(scrollLeft, scrollTop, scrollTarget) {
```

Comment:
I'm worried that `Polymer.AppLayout.scrollTo` is too similar to `window.scrollTo` (which of course will do scroll effects), IMO.